tests/Endpoints/Reports/AccountListingWithListingPriceTest.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+namespace Hitmeister\Component\Api\Tests\Endpoints\Reports;
+
+use Hitmeister\Component\Api\Endpoints\Reports\AccountListingWithListingPrice;
+use Hitmeister\Component\Api\Tests\TransportAwareTestCase;
+
+class AccountListingWithListingPriceTest extends TransportAwareTestCase
+{
+	public function testInstance()
+	{
+		$post = new AccountListingWithListingPrice($this->transport);
+		$this->assertEquals([], $post->getParamWhiteList());
+		$this->assertEquals('POST', $post->getMethod());
+		$this->assertEquals('reports/account-listing-with-listing-price/', $post->getURI());
+	}
+}
```

Comment:
Please add a new line at the end of this file as well

Reply:
added new line at the end